ros2bag/test/test_record_qos_profiles.py

```diff
@@ -72,7 +72,7 @@ def launch_bag_command(self, arguments, **kwargs):
     def tearDownClass(cls):
         try:
             cls.tmpdir.cleanup()
-        except OSError:
+        except (OSError, PermissionError):
```

Comment:
`PermissionError` is an `OSError`, so technically this shouldn't change anything https://docs.python.org/3/library/exceptions.html#exception-hierarchy

Reply:
Yeah. you are right - trying another workaround by explicitly calling `shutil.rmtree(cls.tmp_dir_name, ignore_errors=True)`